public/assets/base.css

```diff
@@ -42,3 +42,13 @@
   font-weight: 400;
   font-style: normal;
 }
+
+@keyframes spin {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  100% {
+    transform: rotate(360deg);
+  }
+}
```

Comment:
Keyframes should be described in Panda. Why add it straight to css here? Outside of font definitions, there should be no css here.

Reply:
On yesterdays call I said I didn't understand how to do some things with panda yet. Once of them is properly configuring it to provide the `base.css`. Do you have any ideas about how to do that? 

Asking as I was working on this [task](https://github.com/leather-wallet/extension/issues/4398) and came up with this [WIP PR](https://github.com/leather-wallet/extension/pull/4494). 

For that bug:
- when the app is loaded the BG colour flashes white before the them BG colour gets set properly
- we can try set BG to the theme BG but we don't know what it is until Panda provides it
- I was trying to set the colour in the `base.css` and it works but not in all cases

Panda has different [layers](https://panda-css.com/docs/concepts/cascade-layers) but I couldn't understand how to extend and configure them properly. For example, ideally we could have our fonts loaded in the `@layer base` 

Maybe you understand better how that works @kyranjamie ?

Reply:
I did try to import `keyframes` from leather-styles here, but I don't think it worked. I can try it again thou.

Reply:
I see now `spin` is built into panda. 👍

Reply:
Afaik we basically shouldn't ever need to touch the base.css. If we need global styles, panda has it's method for adding these [imagine that's where the dark mode stuff would go](https://panda-css.com/docs/migration/styled-components#global-styles).

Reply:
The layers thing is the only direct css file we need to add re: panda, and it is just to pick and choose which panda-generated styles we wanna add.